modules/rostests/apitests/user32/makeimg/readme.txt

```diff
@@ -0,0 +1,14 @@
+For reference see https://jira.reactos.org/browse/CORE-17005.
+
+This should be compiled using MSVC to create the image.dll file for testing.
+Also, it creates an interactive showimg.exe program that can be used to verify this file.
+After the creation of the image.dll file using MSVC it can be used to create user32_apitest:LoadImageGCC.
+Simply copy the MSVC created 'image.dll' into the 'modules\rostests\apitests\user32' subdirectory to use it.
+This file already exists there so that this step is not necessary, but only presents another option.
+
+Unfortunately, this test will not work correctly using Windows 2003 Server SP2.
```

Comment:
Is there any mechanism that does exclude this test to be run automatically by the WHS-bot? If not, it should be added.

Reply:
I concur.
Moreover, is it S2003 only? What about XP or Vista+?

Reply:
This test does not work on any Windows version that I have tested. I have not had a chance to look into this, but I know that the reason will be complex and maybe more than I am capable of finding. I have added a test for IsReactOS for now that keeps the WHS testman results from showing any negative effects from this new test. Hopefully, this will suffice for the present. See this for example: https://reactos.org/testman/compare.php?ids=94718 Thanks.

Reply:
> This test does not work on any Windows version that I have tested.

I suggest to explicitly write so, instead of pointing to S2003sp2 specifically.

> https://reactos.org/testman/compare.php?ids=94718

The new test is not even reported :-/

> Additionally, the test fails when it gets a NULL returned when trying to get a handle to the bitmap (hBmp)

You may add a code comment to pinpoint where to look for Windows failure.
Side-note: This later investigation might reveal a "bug" in ROS, which succeeds where Windows fails.

Reply:
For Examples of the WHS bot results I gave a bad link above. Please see these:
https://reactos.org/testman/compare.php?ids=94717
https://reactos.org/testman/compare.php?ids=94740

Reply:
gcc produces a broken file,
we try to handle this gracefully, windows does not do this.